Sources/Defaults/Defaults+Extensions.swift

```diff
@@ -0,0 +1,113 @@
+}
+extension String: Defaults.Serializable {
+	public static let isNativelySupportedType = true
+}
+extension CGFloat: Defaults.Serializable {
+	public static let isNativelySupportedType = true
+}
+extension Int8: Defaults.Serializable {
+	public static let isNativelySupportedType = true
+}
+extension UInt8: Defaults.Serializable {
+	public static let isNativelySupportedType = true
+}
+extension Int16: Defaults.Serializable {
+	public static let isNativelySupportedType = true
+}
+extension UInt16: Defaults.Serializable {
+	public static let isNativelySupportedType = true
+}
+extension Int32: Defaults.Serializable {
+	public static let isNativelySupportedType = true
+}
+extension UInt32: Defaults.Serializable {
+	public static let isNativelySupportedType = true
+}
+extension Int64: Defaults.Serializable {
+	public static let isNativelySupportedType = true
+}
+extension UInt64: Defaults.Serializable {
+	public static let isNativelySupportedType = true
+}
+
+extension URL: Defaults.Serializable {
+	public static let bridge = Defaults.URLBridge()
+}
+
+extension Defaults.Serializable where Self: Defaults.CollectionSerializable, Element: Defaults.Serializable {
+	public static var bridge: Defaults.CollectionBridge<Self> { Defaults.CollectionBridge() }
+}
+
+extension Defaults.Serializable where Self: Defaults.SetAlgebraSerializable, Element: Defaults.Serializable & Hashable {
+	public static var bridge: Defaults.SetAlgebraBridge<Self> { Defaults.SetAlgebraBridge() }
+}
+
+extension Defaults.Serializable where Self: Codable {
+	public static var bridge: Defaults.TopLevelCodableBridge<Self> { Defaults.TopLevelCodableBridge() }
+}
+
+extension Defaults.Serializable where Self: RawRepresentable {
+	public static var bridge: Defaults.RawRepresentableBridge<Self> { Defaults.RawRepresentableBridge() }
+}
+
+extension Defaults.Serializable where Self: RawRepresentable & Codable {
+	public static var bridge: Defaults.RawRepresentableCodableBridge<Self> { Defaults.RawRepresentableCodableBridge() }
+}
+
+extension Defaults.Serializable where Self: NSSecureCoding {
+	public static var bridge: Defaults.NSSecureCodingBridge<Self> { Defaults.NSSecureCodingBridge() }
+}
+
+extension Set: Defaults.Serializable where Element: Defaults.Serializable {
+	public static var bridge: Defaults.SetBridge<Element> { Defaults.SetBridge() }
+}
+
+extension Optional: Defaults.Serializable where Wrapped: Defaults.Serializable {
+	public static var isNativelySupportedType: Bool { Wrapped.isNativelySupportedType }
+	public static var bridge: Defaults.OptionalBridge<Wrapped> { Defaults.OptionalBridge() }
+}
+
+extension Array: Defaults.Serializable where Element: Defaults.Serializable {
+	public static var isNativelySupportedType: Bool { Element.isNativelySupportedType }
+	public static var bridge: Defaults.ArrayBridge<Element> { Defaults.ArrayBridge() }
+}
+
+extension Dictionary: Defaults.Serializable where Key == String, Value: Defaults.Serializable {
```

Comment:
```suggestion
extension Dictionary: Defaults.Serializable where Key: ExpressibleByStringLiteral, Value: Defaults.Serializable {
```

?

Reply:
In [swift-corelibs-foundation](https://github.com/apple/swift-corelibs-foundation/blob/ef6f96ee82ea0f54252071c0ecadf5f01be9aecc/Foundation/UserDefaults.swift#L41), it seems that `Key` must be a String type?

Reply:
I think using `LosslessStringConvertible & Hashable` is more comprehensive.
We can initialize dictionary's `Key` by `String` with `init?(_ description: String)`
And initialize `String` by dictionary's `Key` with `init<T>(_ value: T) where T : LosslessStringConvertible`